--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -131,7 +131,7 @@

 include(CheckCCompilerFlag)
 check_c_compiler_flag(-Wall HAS_WALL)
-set(COGCORE_INCLUDE_DIRS ${WEB_ENGINE_INCLUDE_DIRS} ${SOUP_INCLUDE_DIRS})
+set(COGCORE_INCLUDE_DIRS ${WEB_ENGINE_INCLUDE_DIRS} ${SOUP_INCLUDE_DIRS} /usr/include/wpe-1.0 /usr/include/wpe-fdo-1.0 /usr/include/wpe-webkit-1.0)
 set(COGCORE_CFLAGS ${WEB_ENGINE_CFLAGS_OTHER} ${SOUP_CFLAGS_OTHER})
 if (HAS_WALL)
   set(COGCORE_CFLAGS ${WEB_ENGINE_CFLAGS_OTHER} ${SOUP_CFLAGS_OTHER} "-Wall")
